searchsummary/src/vespa/searchsummary/docsummary/resultclass.cpp

```diff
@@ -54,4 +53,18 @@ ResultClass::AddConfigEntry(const char *name)
     return AddConfigEntry(name, {});
 }
 
+bool
+ResultClass::all_fields_generated(const vespalib::hash_set<vespalib::string>& fields) const
+{
+    if (fields.empty()) {
+        return _dynInfo._generateCnt == GetNumEntries();
```

Comment:
Consider checking this first even when fields is not empty. If all fields in the result class are generated then any subset of those fields will also all be generated.

Reply:
Fixed.